spec/features/visit_home_page_spec.rb

```diff
@@ -10,6 +10,43 @@
     expect(page).to have_text "sign up here"
   end
 
+  context "for the www.getyourrefund.org domain" do
+    before do
+      Capybara.default_host = "https://www.getyourrefund.org"
```

Comment:
I'm worried this will leak into other tests. What if we omit this entirely, and stick to the test host?

> The main reason is that changing the default_host or app_host option will change this value globally for each spec that follows. The author of Capybara, Jonas Nicklas, states "Never change Capybara.default_host, just don't." and recommends that subdomains are visited explicitly."

https://minimul.com/capybara-and-subdomains.html

Reply:
yikes good catch!